src/staging/src/components/pageContent/pageContent.js

```diff
@@ -22,12 +23,32 @@ export default function PageContent({ contentType, content }) {
       return (
         <LinkCard
           title={content.title}
+          subTitle={content.subTitle}
           url={content.url}
           imageUrl={content.imageUrl}
           imageAltText={content.imageAltText}
+          variation={content.variation}
+          buttonText={content.buttonText}
         ></LinkCard>
       )
     }
+  if (contentType === "parks.card-set") {
+    return (
+      <div className="row">
+        {content.cards.map(card =>
+          <LinkCard
```

Comment:
Missing `key` prop on the list of `LinkCard` components.  You should see the "missing key prop" errors in your console

Reply:
indeed, thx.